src/utils/index.ts

```diff
@@ -1,21 +1,22 @@
-// import * as dotenv from 'dotenv';
 import * as fs from 'fs';
 import * as path from 'path';
 import { TwilioServerlessApiClient } from '@twilio-labs/serverless-api';
 import * as twilio from "twilio"; 
 import { getAPI } from './api';
 import { isEqual } from 'lodash';
 
-//TODO: check incompatibility of using dotenv
-// export const getEnv = (path:string) => {
+export const getEnv = (path:string) => {
  
-//   if (fs.existsSync(path)) {
-//      return dotenv.parse(fs.readFileSync(path, 'utf8').toString())
-//   }
+  if (fs.existsSync(path)) {
+
+    const dotenv = require('dotenv').config({ path: path });
```

Comment:
I would move the `require()` on the top of the file (nice to have)

Reply:
Yep, it is related to that. Not sure what is happening. I've tried to install other libraries but they fail with the same error. I think it is related to how Pulumi runs the code.